Userland/Utilities/lsusb.cpp

```diff
@@ -18,15 +19,58 @@
 #include <stdio.h>
 #include <unistd.h>
 
+// FIXME: Hmmm, perhaps this should be part of the USBDB library?
```

Comment:
I think what should happen is:

- Rename `LibUSBDB` to `LibUSB`, expose its `Database` as `USB::Database`
- Move this there

Reply:
Good idea, it's probably good to have that as part of our (eventual) userspace lib.

Reply:
I'm removing this, we already have support for this in `LibUSBDB`